spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/functions/SparkFunctions.java

```diff
@@ -30,14 +30,19 @@ public class SparkFunctions {
   private SparkFunctions() {}
 
   private static final Map<String, UnboundFunction> FUNCTIONS =
-      ImmutableMap.of(
-          "iceberg_version", new IcebergVersionFunction(),
-          "years", new YearsFunction(),
-          "months", new MonthsFunction(),
-          "days", new DaysFunction(),
-          "hours", new HoursFunction(),
-          "bucket", new BucketFunction(),
-          "truncate", new TruncateFunction());
+      new ImmutableMap.Builder<String, UnboundFunction>()
+          .put("iceberg_version", new IcebergVersionFunction())
+          .put("years", new YearsFunction())
+          .put("year", new YearsFunction())
```

Comment:
We could extract a constant instead of initializing two instances. 
(This comment might be conflicted with my another comment in TestSparkDaysFunction)